docs/www/acls.md

```diff
@@ -56,61 +62,160 @@ redpanda:
+
+You config must look something like this:
 
 ```yaml
 redpanda
   
   enable_sasl: true
   superusers:
-  - admin
+    - username: admin
   # The rest of the config...
 ```
 
+<tabs>
+  <tab id="Local Redpanda">
+  You can find the configuration file here
+
+  ```bash
+  /etc/redpanda/redpanda.yaml
+  ```
+
+After you change it's value, restart Redpanda service for changes to be in effect. 
+
+  </tab>
+  <tab id="Docker Container">
+  To access the files inside the container, first you have to enter the container.
+
+  You can do that by running:
+
+```bash
+  docker exec -it <name-of-container> bash
+```
+
+Change <name-of-container> for your container and execute the command. 
+Then go to the same directory as a local redpanda config:
+
+```bash
+  /etc/redpanda/redpanda.yaml
+```
+
+After you finish it, restart the container for changes to be in effect. 
+
+  </tab>
+  <tab id="Kubernetes">
+  For Kubernetes, things are a little different. You can change your configuration file with the YAML for the cluster.
+
+  If you got our [one-cluster sample](https://github.com/vectorizedio/redpanda/blob/dev/src/go/k8s/config/samples/one_node_cluster.yaml) you should add these values to enable different flags.
+
+  For example: 
+
+```yaml
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+  name: external-connectivity
+spec:
+  image: "vectorized/redpanda"
+  version: "latest"
+  replicas: 3
+  enableSasl: true
+  superUsers:
+    - username: admin
+  resources:
+    requests:
+      cpu: 1
+      memory: 2Gi
+    limits:
+      cpu: 1
+      memory: 2Gi
+  configuration:
+    rpcServer:
+      port: 33145
+    kafkaApi:
+     - port: 9092
+     - external:
+         enabled: true
+    pandaproxyApi:
+     - port: 8082
+     - external:
+         enabled: true
+    adminApi:
+    - port: 9644
+    developerMode: true
+```
+
+**_Note_** - The attributes in K8s YAML are in camelCase instead of snake_case used in local redpanda.yaml
+
+Remember, that after you change the config file you **must restart** the pods for changes to be in effect.
+
+You can check if your spec is correct by running: 
+
+```
+kubectl get clusters external-connectivity -o=jsonpath='{.spec}' 
+```
+
+  </tab>
+</tabs>
+
+
 Then, set a password for the user by running the following command. Replace `<password>` for a password of your choice. 
 
-```cmd
-$ rpk acl user create \
-  --new-username admin \
-  --new-password <password> \
-  --api-urls localhost:9644
+```bash
+rpk acl user create \
+--new-username admin \
+--new-password <password> \
+--api-urls localhost:9644
+```
+
+The response should be:
 
+```
 Created user 'admin'
 ```
 
+**_Note_** - `rpk acl` commands are defaulted to connect to `localhost:9644`. If you have different configuration, you will need to pass `api-urls <node IP>:<admin API port>`
```

Comment:
- `>` indents the note
- Avoid "will"

```suggestion
> **_Note_** - `rpk acl` commands are defaulted to connect to `localhost:9644`. If you use a different port, pass `api-urls <node IP>:<admin API port>`
```

Reply:
I really mean configuration here, because if the guy sets a domain for example, he will need to change both node IP and admin port.

Reply:
Maybe "IP address, URL, or port..."